[IMP] Employees: Update departments (#13055)
odoo/documentation: content/applications/hr/employees/departments.rst

  parent department), select the parent department using the drop-down menu.
- :guilabel:`Company`: using the drop-down menu, select the company the department is part of. This
  field only appears in a multi-copany database.
- :guilabel:`Color`: select a color for the department. Click the colored box to display all the
  color options. Click on a color to select it.
- :guilabel:`Appraisal Templates`: using the drop-down menu, select the appraisal form to be used
  for all employee appraisals within this department. If a new appraisal is desired, enter the name
  for the appraisal, then click :guilabel:`Create and edit...` to modify the new appraisal form.
  This field **only** appears if the **Appraisals** app is installed.
- :guilabel:`Appraisal Survey`: using the drop-down menu, select the default survey to use for the
  department when requesting feedback from employees within the department. The default options are
  :guilabel:`Employee Opinion Form`, :guilabel:`360 Feedback`, and :guilabel:`Employee Appraisal Form`.
  This field **only** appears if the **Appraisals** app is installed, *and* the *360 Feedback*

>   option is enabled in the settings.

Reviewer: Just dropping a note here in a random spot 😅 — we usually don’t cover all the smart buttons unless they’re directly relevant to the doc. That said, I am curious about the Plans smart button! Do you happen to cover it in one of your HR docs? If so, it could be super helpful to link to that in a seealso block at the end of this section 😊

![image](https://github.com/user-attachments/assets/8f4bcdfe-0812-4561-aa43-055bf801d6ca)

After the form is completed, click the :icon:`fa-cloud-upload` :guilabel:`(cloud upload)` icon to
manually save the changes. When saved, a :guilabel:`DEPARTMENT ORGANIZATION` chart appears in the
top-right of the department card, illustrating where the departmen lies in the organization.

.. image:: departments/department-form.png
   :alt: The department for with all fields filled out.

.. note::

>    The form auto-saves while data is entered, however the :guilabel:`Department Organization` chart

Reviewer: I cant comment on the seealso line on 45, so I'm doing it here!

There’s no hard rule for this, but just as a heads-up—we usually keep seealso blocks link-only, without any extra text. Something like this is perfect:

.. seealso::
   :doc:`../appraisals`

  department :ref:`awaiting approval <time_off/manage-allocations>`. This **only** appears if there
  are requests to approve.
- :guilabel:`New Applicants`: the number of :ref:`new applicants <recruitment/new>` for a position
  within the department. This **only** appears if there are new applicants.
- :guilabel:`Expense Reports`: the number of employees in the department with :doc:`open expense
  reports to approve <../../finance/expenses/approve_expenses>`. This **only** appears if there are
  any expense reports waiting for approval.

> - :guilabel:`Absence`: the number of employees out on an approved time off request for the current

Reviewer: ```suggestion
- :guilabel:`Absence`: the number of employees with approved time off for the current
```

:guilabel:`Unfold` button changes to a :guilabel:`Fold` button. To collapse the department, click
the :guilabel:`Fold` button. Only **one** department *per row* can be unfolded at a time.

Click anywhere on a department card to open the department form. Click the :guilabel:`(#) Employees`
smart button to view a list of all the employees in that department, including all employees in the
child departments beneath it, organized by individual department.

.. example::
   In the hierarchy view, if the :guilabel:`(2) Employees` button on the :guilabel:`Management` card
   is clicked (the highest-level department card), **all** employees appear in a list view, grouped
   by department. This is because **all** departments are children of the :guilabel:`Management`
   department.

   If the :guilabel:`(3) Employees` button in the :guilabel:`Sales` department card is clicked, the
   employees from the :guilabel:`Sales` department, as well as its two child departments
   (:guilabel:`East Coast Territory` and :guilabel:`West Coat Territory`), appear in the list.

  .. image:: departments/hierarchy.png
     :align: center
     :alt: The departments presented in a hierarchy view.

>   .. image:: departments/employee-list.png

Reviewer 2: The image file (`employee-list.png`) is not used, please remove :)
